controllers/conditions.go

```diff
@@ -183,29 +207,34 @@ func (r *ImageClusterInstallReconciler) setClusterTimeoutConditions(ctx context.
 		Message: message,
 	})
 
+	r.Log.Info("Setting cluster timeout conditions")
```

Comment:
Same here, shouldn't this patch and log be conditional on the calls above?

Reply:
not in this case as we will set this condition once as reconcile will exit in case it was set already